dts/arm/st/l0/stm32l031X6.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/l0/stm32l053.dtsi>
```

Comment:
Sorry for not catching this earlier but this is not correct. This should be <st/l0/stm32l031.dtsi>.
So you need to add st/l0/stm32l031.dtsi.
It could be done by including st/l0/stm32l011.dtsi and adding larger EEPROM.

Reply:
This should now be fixed.